client/pages/project-edit-modal/modal.template.html.slim

```diff
@@ -3,22 +3,16 @@ form#project-edit-modal novalidate=true role='form' name='projectForm' ng-submit
     h3.modal-title ng-if='editedProject.id' Edit {{ project | projectName }}
     h3.modal-title ng-if='!editedProject.id' New Project
   .modal-body
-    .form-group ng-class='{"has-error": projectForm.name.$dirty && projectForm.name.$invalid}'
-      label for='projectName'
+    .form-group ng-class='{"has-error": projectForm.displayName.$dirty && projectForm.displayName.$invalid}'
+      label for='projectDisplayName'
         | Name
         span.required<
           | (required)
-      input#projectName.form-control type='string' name='name' placeholder='Name (used in URLs)' required=true maxlength=50 ng-pattern='/^[a-z0-9]+(?:-[a-z0-9]+)*$/i' unique-project-name=true autofocus=true ng-model='editedProject.name' ng-model-options='{updateOn: "default blur", debounce: {default: 500, blur: 0}}'
+      input#projectDisplayName.form-control type='string' name='displayName' placeholder='Name' maxlength=50 ng-model='editedProject.displayName' unique-project-name=true autofocus=true
       p.help-block
-        | This is the identifier of the project and is used in URLs.
-        |  It can only contain alphanumeric characters and single hyphens (not at the beginning or end).
-      .alert.alert-danger ng-if='projectForm.name.$error.uniqueProjectName'
+        | This is the name of the project as shown in ProbeDock.
```

Comment:
Replace with "The name of the project, unique within the organization, up to 50 characters."